frontend/package.json

```diff
@@ -19,8 +19,8 @@
     "build-test-pa11y": "./development-configurations/build-test-pa11y.sh",
     "snyk-test": "snyk test",
     "start": "ng serve",
-    "test:ci": "ng test --sourcemaps=false --config ./development-configurations/karma.conf --single-run=true --watch=false --progress=false --code-coverage",
-    "test": "ng test --config ./development-configurations/karma.conf --sourcemaps=true",
+    "test:ci": "ng test --karma-config ./development-configurations/karma.conf --single-run=true --watch=false --progress=false --code-coverage",
```

Comment:
It looks like these changes may have broken the e2e tests on circleci, they don't appear to run in the associated test https://circleci.com/gh/18F/fs-permit-platform/1019?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link